geometry_msgs/Point goal
float64 angleDegrees
---
uint8 distance
